src/components/Popup.js

```diff
@@ -3,6 +3,11 @@ import { closeButtons } from "../utils/constants.js";
 export default class Popup {
```

Comment:
`import { closeButtons } from "../utils/constants.js";` 

Please, delete it from line 1

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.



------------------


You need to find the close icon in line 6 only once inside `this._popupElement` and add the listener to it (not **document**)